tests/unit/test_deprecations.py

```diff
@@ -27,9 +27,6 @@ def test_mx7_supported(self):
 
 
 class TestCaseMxMaintained(TestCase):
-    def test_mx7_maintained(self):
-        assert runtime.is_version_maintained(MXVersion("7.23.1"))
-
```

Comment:
Let's move this one to `test_mx7_not_maintained`

Reply:
for extended support we will still support this. Do we want customers to see a line that it is unsupported in that case?

Reply:
Should we add an extended support function / test? 🤔

Reply:
Good idea @jeohist ! @Yornik Let's do it in another PR, revert this chunk and revert https://github.com/mendix/cf-mendix-buildpack/pull/711/files#diff-54130e27ebbe04d07ebe3741504e8adc20f28fa9cd586a5f6d0ad14584be6878L43-L44 as well.

Reply:
reverted